parachain/test-parachains/undying/src/lib.rs

```diff
@@ -90,24 +90,52 @@ pub struct GraveyardState {
 	pub seal: [u8; 32],
 }
 
+/// A structure that configures HRMP messages produced by undying collator
+#[derive(Clone, Encode, Decode, Debug)]
+pub struct HrmpChannelConfiguration {
+	/// Where to send HRMP messages
+	pub destination_para_id: u32,
+	/// Message size
+	pub message_size: u32,
```

Comment:
This interface seems to allow to configure the rate (even if it will be probabilistic) and actual bandwidth, but as state above I think we need to make clear how many messages one para is allowed to send, and the fact that it is actually a blob.